docs/fitting.rst

```diff
@@ -666,6 +666,10 @@ convenience functions to perform exactly this task.  An example is shown below.
     plt.title('Continuum Fitting')
     plt.grid('on')
 
+    # construct normalized spectrum by deviding the old spectrum
```

Comment:
deviding -> dividing

Reply:
@brechmos-stsci : Thanks! Done and force-pushed (this does not warrant a new commit).